source/_posts/2016-08-03-making-page-objects-to-play-nicely-with-ember-wormhole.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Making page objects to play nicely with ember-wormhole"
+category: ember
+comments: true
+author:
+  name: 'Santiago Ferreira'
+  email: 'santiago.ferreira@wyeworks.com'
+  twitter_handle: 'san650'
+  github_handle: 'san650'
+  image: '/images/team/santiago-ferreira.jpg'
+  description: 'Developer working with Ruby, JavaScript and Ember. Love working with Free Software in general. Ember and Angular meetup organizer in Montevideo'
+---
+
+A very common problem people have while testing an ember application that uses the wonderful [ember-wormhole](https://github.com/yapplabs/ember-wormhole) addon is making the page objects to play nicely with it.
```

Comment:
maybe add another sentence like `In this blog post I'll try to cover some of the problems and solutions regarding this.` to complete the introduction to the blog post.